integration-tests/main/src/main/java/io/quarkus/example/rest/TestResource.java

```diff
@@ -295,4 +331,30 @@ public void setValue(String value) {
             this.value = value;
         }
     }
+
+    @Schema()
+    public static class MyOpenApiEntityV1 {
```

Comment:
No need. I wanted to test @Schema scanning when a schema class is the annotation target. Can be removed.

Reply:
OK, so I think testing that is required but... you should do that in a specific test. Otherwise if one part is working but not the other, your test will be successful anyway.

You need one test per feature in this case.